_includes/1.0/overview/security/oauth2/downloads.html

```diff
@@ -0,0 +1,29 @@
+<div id="downloads" class="group">
+
+    <h2>Support Downloads for Development</h2>
+
+    <p>The following downloads can be used by developers to validate the client against OneUp <code>staging</code> environment</p>
+
+    <p>
+        <form method="get" action="{{ site.url }}/downloads/http-client.zip">
+            <button type="submit" class="btn btn-outline-primary">
```

Comment:
Can you improve the presentation? I was clicking the blue link to download the postman collection but I was redirected to their website. It's not clear you have to click the picto.
<img width="517" alt="Screen Shot 2020-12-14 at 5 19 21 PM" src="https://user-images.githubusercontent.com/2098871/102155507-84112300-3e30-11eb-8dd1-097d91013bd1.png">

Reply:
Improved with the commit [908f6b3724d13700528479bf0e305807c8ca9161](https://github.com/myERP/myerp.github.io/pull/31/commits/908f6b3724d13700528479bf0e305807c8ca9161)